fem/gridfunc.hpp

```diff
@@ -726,6 +732,15 @@ inline void QuadratureFunction::GetElementValues(int idx, Vector &values) const
       values(i) = *(q++);
    }
 }
+// fix me: This function should have a more efficient method for doing this operation.
```

Comment:
Yep, please implement the more efficient method. See the last function in this hpp.

Reply:
Thanks for reminding me about that should be fixed now